packages/nextjs/services/web3/connectors.tsx

```diff
@@ -1,15 +1,21 @@
-import { argent, braavos, InjectedConnector } from "@starknet-react/core";
+import {
+  argent,
+  braavos,
+  Connector,
+  InjectedConnector,
+} from "@starknet-react/core";
 import { getTargetNetworks } from "~~/utils/scaffold-stark";
 import { BurnerConnector } from "./stark-burner/BurnerConnector";
 import scaffoldConfig from "~~/scaffold.config";
 import { LAST_CONNECTED_TIME_LOCALSTORAGE_KEY } from "~~/utils/Constants";
+import { controllerInstance } from "~~/utils/scaffold-stark/controller";
 
 const targetNetworks = getTargetNetworks();
 
 export const connectors = getConnectors();
 
 // workaround helper function to properly disconnect with removing local storage (prevent autoconnect infinite loop)
-function withDisconnectWrapper(connector: InjectedConnector) {
+function withDisconnectWrapper(connector: InjectedConnector | Connector) {
```

Comment:
can you check that if you use `Connector` only everything still works? `InjectedConnector` might have been inherited from `Connector`, and if it works, we can remove `InjectedConnector` from here

Reply:
okay

Reply:
It works without it I'll go ahead and remove it